scripts/coderabbit_status.py

```
@@ -0,0 +1,183 @@
+
+def analyze_coderabbit(pr_info: dict, inline_comments: list) -> dict:
+    """Klassifiziert den aktuellen Zustand von CodeRabbit anhand der PR-Kommentare und Checks."""
+    comments = pr_info.get("comments", []) or []
+    checks = pr_info.get("statusCheckRollup", []) or []
+
+    # 1. Prüfen auf Rate-Limit
+    for c in comments:
+        body = c.get("body", "")
+        author = c.get("author", {}).get("login", "")
+        if "coderabbit" in author.lower() or "coderabbit.ai" in body:
+            if "rate limited by coderabbit.ai" in body or "Review limit reached" in body:
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)